astropy/coordinates/baseframe.py

```diff
@@ -495,8 +495,9 @@ def __init__(self, *args, copy=True, representation_type=None,
                 if repr_kwargs.get('distance', True) is None:
                     del repr_kwargs['distance']
 
-                if (issubclass(representation_cls, r.SphericalRepresentation)
-                        and 'distance' not in repr_kwargs):
```

Comment:
I think this is the current recommendation (start the line with the operator rather than end with it), but then my editor complains for both solutions...

Reply:
Hm, my editor complains if I do the opposite here.

Reply:
I think when we make the PEP 8 check very strict for the repo, this operator before/after check is one that I would be very happy to permanently ignore. For now, I think a trailing `# noqa` comment would ignore it?

Reply:
oh, for sure. The other that is flexible is the white space around operators. In many cases those don't make much sense either.

out of curiosity, @adrn would your editor respect an ignore list in e.g. setup.cfg of the repo?

Reply:
Yeah I agree on reverting the change here